src/main/java/io/nats/bridge/jms/support/JMSMessageBusBuilder.java

```diff
@@ -328,8 +403,22 @@ public JMSMessageBusBuilder withSessionCreator(final Function<Connection, Sessio
 
     public MessageBus build() {
         return new JMSMessageBus(getDestination(), getSession(), getConnection(), getResponseDestination(),
-                getResponseConsumer(), getTimeSource(), getMetrics(), getProducerSupplier(), getConsumerSupplier(), getMetricsProcessor(), getTryHandler(), getJmsBusLogger());
+                getResponseConsumer(), getTimeSource(), getMetrics(), getProducerSupplier(), getConsumerSupplier(),
+                getMetricsProcessor(), getTryHandler(), getJmsBusLogger(), getJmsReplyQueue(), getJmsMessageConverter(), getBridgeMessageConverter());
     }
 
 
+    public Hashtable<String, Object> getJndiProperties() {
+        if (jndiProperties.size() == 0) {
+            jndiProperties.put("java.naming.factory.initial", (String) System.getenv().getOrDefault("NATS_BRIDGE_JMS_NAMING_FACTORY", "org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory"));
+            jndiProperties.put("connectionFactory.ConnectionFactory", (String) System.getenv().getOrDefault("NATS_BRIDGE_JMS_CONNECTION_FACTORY", "tcp://localhost:61616"));
```

Comment:
There was a requirement to support the MQSERVER, MQCHLLB, and MQCHLTAB variables.  Would these fall into a custom connection factory?

Reply:
environment vars? We will have a config file.